### PR TITLE
Update DigitalOcean how-to after v0.17

### PR DESCRIPTION
**Pull request by eskombro**
Since v0.17.0, MeiliSearch DigitalOcean droplet won't show the front-end interface if the environment is set to production. The screenshot in the `how-to` guide is replaced by an example on calling the `/health` route.

### CODE REVIEW

**Comment by eskombro** [comment]
Can you also review please @curquiza ? :)

File: resources/howtos/digitalocean_droplet.md [hunk 1]
```diff
@@ -105,6 +105,20 @@ A script will run automatically, asking for your settings and desired configurat
 
 ### 3. Enjoy your ready-to-use MeiliSearch droplet
 
-![Enjoy](/digitalocean/13.finish.png)
+Your MeiliSearch droplet is ready to be used in production.
+
+To check if everything is running smoothly, do an HTTP call to the health route:
+
+```bash
+curl -v https://<your-meilisearch-url>/health
```
Comment (curquiza) [inline resources/howtos/digitalocean_droplet.md:113]:
```suggestion
$ curl -v https://<your-meilisearch-url>/health
```

File: resources/howtos/digitalocean_droplet.md [hunk 2]
```diff
@@ -105,6 +105,20 @@ A script will run automatically, asking for your settings and desired configurat
...
+```
+
+The server should answer with a `204 No content` ststus code as shown in teh example below:
```
Comment (curquiza) [inline resources/howtos/digitalocean_droplet.md:116]:
```suggestion
The server should answer with a `204 No content` status code as shown in the example below:
```

**Review by curquiza — CHANGES_REQUESTED** [review]

File: resources/howtos/digitalocean_droplet.md [hunk 3]
```diff
@@ -105,6 +105,20 @@ A script will run automatically, asking for your settings and desired configurat
 
 ### 3. Enjoy your ready-to-use MeiliSearch droplet
 
-![Enjoy](/digitalocean/13.finish.png)
+Your MeiliSearch droplet is ready to be used in production.
```
Comment (curquiza) [inline resources/howtos/digitalocean_droplet.md:108]:
```suggestion
Your MeiliSearch Droplet is ready to be used in production.
```

**Review by curquiza — APPROVED** [review]

File: resources/howtos/digitalocean_droplet.md [hunk 4]
```diff
@@ -105,6 +105,20 @@ A script will run automatically, asking for your settings and desired configurat
 
 ### 3. Enjoy your ready-to-use MeiliSearch droplet
```
Comment (eskombro) [inline resources/howtos/digitalocean_droplet.md:106]:
```suggestion
### 3. Enjoy your ready-to-use MeiliSearch Droplet
```

**Review by curquiza — APPROVED** [review]

**Review by react-learner — APPROVED** [review]
LGTM!

**Pull request merged**